accounts/src/main/java/eu/kevin/accounts/bankselection/BankSelectionState.kt

```diff
@@ -9,6 +10,7 @@ import kotlinx.parcelize.Parcelize
 @Parcelize
 data class BankSelectionState(
     val selectedCountry: String = "",
-    val supportedBanks: List<Bank> = emptyList(),
+    val bankListItems: List<BankListItem> = emptyList(),
+    val banks: List<Bank> = emptyList(),
```

Comment:
bad code smell when I see **bankListItems** + **banks** in the state. also, **banks** are not used in the view.

Reply:
I did this, because I need to return bank object from BankSelectionFragment. Posible fix would be to leave list<Bank> and convert it to BankListItem inside view.